container/docker/docker.go

```diff
@@ -161,3 +178,21 @@ func parseDockerVersion(full_version_string string) ([]int, error) {
 	}
 	return version_array, nil
 }
+
+// TODO: switch to a semantic versioning library.
```

Comment:
This TODO is creating confusion, lemme remove it. Any of the docker versions are not following semver format([Refer Docker Issue 31075](https://github.com/docker/docker/pull/31075#issuecomment-280619070)) so it does not make sense to use any prebuilt semver library anyways.